docs/modules/ROOT/content-nav.adoc

```diff
@@ -55,6 +55,7 @@
 ***** xref:ogm/api-reference/model/delete.adoc[]
 ***** xref:ogm/api-reference/model/count.adoc[]
 ***** xref:ogm/api-reference/model/aggregate.adoc[]
+** xref:introspector.adoc[Introspect Neo4j schema]
```

Comment:
As you can see from this file, all other entries in this list take the page header as the entry name. Should we maybe simplify the page header instead? I think we should just run with the branding "Introspector".

Reply:
Perhaps the OGM and Introspector should live under root-level page, "Tools", "Additional Tools" or "Additional Tooling"? Open to other suggestions also.